src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
```

Comment:
I think it's better that you updated this line

Reply:
I did